tarantool/dbapi.py

```diff
@@ -0,0 +1,307 @@
+# -*- coding: utf-8 -*-
+import re
+
+from tarantool.error import InterfaceError
+from .connection import Connection as BaseConnection
+
+
+update_insert_pattern = re.compile(r'^UPDATE|^INSERT', re.IGNORECASE)
+
```

Comment:
We have two options:
- use external package like [dsnparse](https://pypi.org/project/dsnparse/)
- write the same util just for our case
What solution do you recommend to choose?

Reply:
> I would also add `use_list` to the optional parameters as we need it to Django.

I don't mind, but please describe its effect. I guess it will affect both a result set container (what is returned from `fetchmany()` and `fetchall()`) as well as a tuple container (`fetchone()` result and elements of `fetch{many,all}()` result). In future it may also affect `<cursor>.description` container.

To be honest, it looks hacky for me. From API point of view something like `row_container` would look better. However I cannot propose an ideal API right now (considering usage, performance and amount of necessary work), so I don't mind including this parameter for now. At least, it solves an actual problem.

> Do you mind if I file an issue and we implement it in the further release?

After brief discussion with Artem we agreed that it worth to implement `connect()` module function with the proposed parameters (and its order), but left `dsn` non-implemented for a while (raise a proper error when it is passed). Of course, an issue should be filed to implement it later.

> What solution do you recommend to choose?

I guess it would be easier to write our own implementation with compliance to the net.box URI format rather than adopt an external one. Everything is quite straightforward for TCP, but Unix sockets may require a custom logic. See [this change](https://github.com/tarantool/tarantool/commit/97918db3673a646ddc3363c5bce4837be026054e) for example.

Reply:
Please see the issue created by the comment above https://github.com/tarantool/tarantool-python/issues/170